.changeset/perfect-drinks-teach.md

```diff
@@ -0,0 +1,19 @@
+---
+'@commercetools-backend/express': minor
+---
+
+Add support for setting the `audience` policy. The policy can be used to determine how the `audience` value is exchanged between the Merchant Center API and the external API.
+
+Supported values are:
+
+- `forward-url-full-path`: This is the default policy. It sets the `audience` using the full URL (origin + pathname).
+- `forward-url-origin`: This is the alternative policy. It sets the `audience` using only the origin URL part.
+
+```js
+createSessionAuthVerifier({
+  audiencePolicy: 'forward-url-origin',
+  // ...
+});
+
+// "X-Forward-To-Audience-Policy": "forward-url-origin"
```

Comment:
Not sure about this header example since it made sense in the `application-shell` changeset because that's the header attached to the request with the context created by the helper but here it seems less clear.
Maybe we want to say that these backend helpers will use that header?

Reply:
True, good catch!

Reply:
```suggestion
```